stories/index.js

```diff
@@ -20,15 +29,30 @@ storiesOf('Range Components/SingleRange', module)
   }))
   .add('with title', () => ({
     components: { BaseSingleRange },
-    template: '<base-single-range :subProps="{title: \'Books Filter\'}"/>',
+    methods: { text:(x,y) => text(x,y)
+    },
+    template: '<base-single-range :subProps="{ title: text(\'title\',\' Books Filter \')}"/>',
   }))
   .add('with defaultSelected', () => ({
     components: { BaseSingleRange },
-    template: '<base-single-range :subProps="{defaultSelected: \'Rating 3 to 4\'}"/>',
+    methods: { text:(x,y) => text(x,y)
+    },
+    template: '<base-single-range :subProps="{defaultSelected: text(\'defaultSelected\',\'Rating 3 to 4\')}"/>',
   }))
+   .add('with filter', () => ({
+    components: { BaseSingleRange },
+    methods: { 
+    text:(x,y) => text(x,y),
```

Comment:
@divyanshu-rawat Can you try to register all knobs methods in Vue global mixings so they'll be available for all components without defining every time in components.
For ref. Check https://stackoverflow.com/questions/42613061/vue-js-making-helper-functions-globally-available-to-single-file-components

Reply:
@bietkul Thanks will work on it 👍

Reply:
@bietkul please check it now I have added mixins.js to access methods in all components.